protocol/contracts/farm/facets/DiamondCutFacet.sol

```diff
@@ -23,7 +24,7 @@ contract DiamondCutFacet is IDiamondCut {
         FacetCut[] calldata _diamondCut,
         address _init,
         bytes calldata _calldata
-    ) external override {
+    ) external override withSeraph() {
```

Comment:
should we remove the function call on `withSeraph` like we did on the other protected functions after updating the import statement?